srl/pom.xml

```diff
@@ -0,0 +1,165 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <parent>
+        <artifactId>illinois-cogcomp-nlp</artifactId>
+        <groupId>edu.illinois.cs.cogcomp</groupId>
+        <version>3.0.73</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>illinois-srl</artifactId>
+    <packaging>jar</packaging>
+    <url>http://cogcomp.cs.illinois.edu</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <cogcomp-nlp-version>3.0.59</cogcomp-nlp-version>
+        <cogcomp-nlp-pipeline-version>0.1.24</cogcomp-nlp-pipeline-version>
+    </properties>
+
+    <dependencies>
+        <!-- Include the pre-trained SRL models for running SemanticRoleLabeler -->
+        <!-- Notice that the models need to match up to the minor version number -->
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-srl-models</artifactId>
+            <classifier>verb-stanford</classifier>
+            <version>5.1</version>
+        </dependency>
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-srl-models</artifactId>
+            <classifier>nom-stanford</classifier>
+            <version>5.1</version>
+        </dependency>
+
+        <!--The Illinois pipeline can be used instead -->
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-nlp-pipeline</artifactId>
+            <version>${cogcomp-nlp-pipeline-version}</version>
+            <exclusions>
+                <exclusion>
+                    <artifactId>illinois-srl</artifactId>
+                    <groupId>edu.illinois.cs.cogcomp</groupId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <!-- The following 3 projects are now developed under illinois-cogcomp-nlp -->
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-core-utilities</artifactId>
+            <version>${cogcomp-nlp-version}</version>
```

Comment:
These `cogcomp-nlp-version` variables should be replaced with the current super-project version.